wasabi/research-club/anonymous-credentials.md

```diff
@@ -44,56 +50,54 @@ I think Peter Branz, not quite sure.
 He wrote his PhD.
 Yeah.
 
-Speaker 1: 00:03:20
+Adam Fiscor: 00:03:20
 
 Stefano, I think, not Peter or something like that.
-Okay.
 
-Speaker 0: 00:03:24
+Jonas Nick: 00:03:24
 
-He wrote his PhD thesis.
 Okay, Branz is his last name.
```

Comment:
Branz is incorect, Brands is the correct lastname of "Mr Stefan Brands". Stefan Brands is the designer of the core cryptographic protocols of Microsoft's U-Prove